lib/srv/sess.go

```diff
@@ -1367,8 +1299,11 @@ func (s *session) exportPartyMembers() []rsession.Party {
 
 // exportParticipants returns a list of all members that joined the party.
 func (s *session) exportParticipants() []string {
-	s.mu.Lock()
-	defer s.mu.Unlock()
```

Comment:
Why did you remove this?

Reply:
It's called under lock in `emitSessionEndEvent`. I'll move this logic there since it's only called once.